### DISCUSSION TITLE
Is there any helm chart for lavinmq?

### DISCUSSION THREAD

**Discussion by ThePlenkov** [cloudamqp/lavinmq#602] [General]
Hi! I'd like to explore LavinMQ by installing to our Kubernetes cluster. Do you know any chart may be? Thanks!

**Comment by viktorerlingsson** [comment]
Hi, 
We don't have a Helm chart for LavinMQ yet, but it's something we probably can fix. I've added it to the backlog.

**Comment by fuad-daoud** [comment]
Hello do you still not have any chart for LavinMQ?

**Reply by viktorerlingsson** [reply]
Hi, 

No, unfortunately we haven't gotten around to doing that yet.

**Comment by prom3theu5** [comment]
I've cobbled one together actually for a project I am using it on
https://github.com/SimCubeLtd/lavinmq-helmchart
